timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -333,7 +334,7 @@ def save(self) -> None:
 
     def _add_model(
         self,
-        model: Union[AbstractTimeSeriesModel, TimeSeriesEnsembleWrapper],
+        model: AbstractTimeSeriesModel,
         base_models=None,
     ):
```

Comment:
Can we add a docstring (and maybe a type annotation) here to explain the meaning of `base_models`? Is this a list of names of models that `model` depends on?

Reply:
Also say that `base_models` should only be provided if `model` is an ensemble model (as checked by the assert).